src/lib/Makefile.am

```diff
@@ -20,7 +20,6 @@ librnp_la_SOURCES	= \
 	key_store.c \
 	key_store_pgp.c \
 	key_store_kbx.c \
-	key_store_ssh.c \
```

Comment:
Ok, I thought that we are at least disabling SSH key code for now. Actually nobody answered in duscussion regarding the SSK keys on Ribose.
Will fix this.